ci-operator/config/openshift/tektoncd-pipeline/OWNERS

```diff
@@ -3,6 +3,25 @@
 # See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
 
 approvers:
-- build-pipeline-approvers
```

Comment:
I wonder why this changes ? (seems unrelated to the PR)
Also bring a question on, shouldn't the others use the same (instead of copying the list on each file) ?

Reply:
I am following what other teams did and there is no OWNERS_ALIAS but I can add it